code/_onclick/observer.dm

```diff
@@ -14,7 +14,7 @@
 
 	// Otherwise jump
 	else if(A.loc)
-		loc = get_turf(A)
+		Move(get_turf(A.loc))

```

Comment:
Wait @Cyberboss this is in the observer code what are you talking about with clients